Laravel.gitignore

```diff
@@ -1,5 +1,7 @@
 vendor/
 node_modules/
+.rocketeer/
```

Comment:
@gaaarfild I'm fine with handling multiple changes to a file in this PR, I'd just love some context on what these folders represent - and links to the relevant projects in case others want to know more...